build/yaml/crd/vpc/crd.nsx.vmware.com_subnets.yaml

```diff
@@ -166,6 +179,11 @@ spec:
                 items:
                   type: string
                 type: array
+              shared:
```

Comment:
in VMODL we call it `assigned subnet`, do we need to keep the same name?
Or we use annotation to identify whether the subnet is shared